socialdistribution/api/views.py

```diff
@@ -830,6 +831,69 @@ def author_profile(request, author_id):
     }
     return JsonResponse(response_body, status=405)
 
+@check_auth
+def github_posts(request, author_id):
+    if request.method == "GET":
+        author = Author.objects.get(id=author_id)
```

Comment:
I think we should get the github posts for the user making the request, `request.user`, since if we do it this way you could update the activity for any user as long as you're logged in.

Reply:
Makes sense, fixed! : )